webapp/src/components/kanban/kanbanCard.tsx

```diff
@@ -16,9 +16,11 @@ import {useSortable} from '../../hooks/sortable'
 
 import './kanbanCard.scss'
 import PropertyValueElement from '../propertyValueElement'
+import {CardTree} from '../../viewModel/cardTree'
 
 type Props = {
     card: Card
+    cardTree: CardTree | undefined
```

Comment:
I think you can just do ? operator if it can also be undefined. You don't have to be explicit.
```suggestion
    cardTree?: CardTree
```

https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties

You can also apply to this  to anywhere else you may have done a `property | undefined` which I may have missed.

Reply:
I was actually using the `?:` in some places but not concisely. I'll keep this in mind.